aii-dhcp/src/main/pan/quattor/aii/dhcp/config.pan

```diff
@@ -2,24 +2,31 @@
 # ${developer-info}
 # ${author-info}
 # ${build-info}
-template quattor/aii/${project.artifactId}/config;
+template quattor/aii/dhcp/config;
 
-include 'quattor/aii/${project.artifactId}/schema';
+include 'quattor/aii/dhcp/schema';
 
-# TFTP server
-# This is optional. Only it is necessary if your TFTP server is running on
-# a different machine than the DHCP server
-#
-# "/system/aii/dhcp/options/tftpserver" = "tftp.mydomain.org"
-#
+bind "/system/aii/discovery/dhcp" = structure_dhcp_module_info;
```

Comment:
fyi, since you do this, there's no way to unbind so the code has to be able to disable the discovery

Reply:
euhm, my point is that this bind disables dhcp configuration and only configures discovery; and there's no way to disable it.
so the template with this defined should just be called `dhcp/discovery.pan`, `dhcp/config.pan` at least suggest it does something with dhcp.

Reply:
Well, `ks/config.pan` and `pxelinux/config.pan` are not called `ks/osinstall.pan` and `pxelinux/nbp.pan` either :-)

The current template layout is inconsistent - it mixes core bits and plugins together. Maybe the templates of real plugins should be moved to a `plugins` subdirectory?

Reply:
my point wrt renaming `dhcp/config` to `dhcp/discvovery` is that the current template and code does not configure dhcp since it enables the discovery plugin/whatever.

i'm aware that it's a mess (and i clearly haven't looked at the code base for a while), but this PR breaks our setup, so i'm trying to figure out where it goes wrong and we can get out of this.